src/pages/ResearcherProfile.js

```diff
@@ -12,6 +12,18 @@
 import { Notification } from '../components/Notification';
 import * as ld from 'lodash';
 import { USER_ROLES, setUserRoleStatuses } from '../libs/utils';
+import {getNames} from "country-list";
+
+const USA = option({ value: "United States of America"}, ["United States of America"]);
+const NA = option({ value: "N/A"}, ["N/A"]);
+const countries = getNames();
+const countryNames = countries.map(name => option({value: name}, [name]));
+countryNames.splice(232, 1);
+countryNames.splice(0, 0, USA);
+const UsaStates = require('usa-states').UsaStates;
+const states = (new UsaStates().arrayOf("names"));
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=6949935)
```suggestion
const stateNames = (states.map((name) => option({value: name}, [name])));
```